modules/test/playwright/tests/object/modelBuilder.spec.ts

```diff
@@ -59,6 +63,8 @@ test('can create relationship by dragging node handles', async ({
 		objectDefinition2.name
 	);
 
+	await _modelBuilderPage.clickFitViewButton();
```

Comment:
The second call to `clickFitViewButton` is not needed to fix this bug, but because the node is taller after showing all fields, I am afraid it will get out of screen somehow if anything changes in the future, so I call it again.

Reply:
I believe a user would have to do this in a real use case. So, that's fine! Thank you for the warning!